CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # CHANGELOG
 
+## Version 0.3.0 - 2024-05-28
+
+### Features
+
+- Adding pipeine for creating dataset for disconnection-aware Chemformer model.
+- Fine-tuning of Chemformer has been updated to support Hydra configuration.
```

Comment:
Add a "bug fix" - "fixing issue in template-pipeline when sibling reaction information is not available"